catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliConstants.java

```diff
@@ -359,4 +361,12 @@
   public static final String PROP_PORT = "PORT";
 
   public static final String UNKNOWN = "Unknown";
+
+  public static Set<String> getContentStrings() {
+    return CONTENT_STRINGS;
```

Comment:
❗️ Both of these were and are still mutable collections. They should be created immutable or wrapped (on creation) as unmodifiable.

Reply:
@coyotesqrl good catch. I wasn't sure if we had a "standard" way of doing it. After talking with @clockard I went with Collections.unmodifiableSet.

Reply:
Actually, I might look at making the static set unmodifiable globally, rather than returning a new one in each method call.